deployment/constructs/waf.ts

```diff
@@ -16,142 +13,141 @@ interface WafProps {
 }
 
 export class WafConstruct extends Construct {
-  public readonly webAcl: CfnWebACL;
-  public readonly logGroup: LogGroup;
+  public readonly webAcl: CfnWebACL | undefined;
+  public readonly logGroup: LogGroup | undefined;
 
   constructor(
     scope: Construct,
     id: string,
     props: WafProps,
-    scopeType: string,
+    scopeType: string
   ) {
     super(scope, id);
 
-    const {
-      name,
-      blockRequestBodyOver8KB = true
-    } = props;
+    if (process.env.CDK_DEFAULT_ACCOUNT !== "000000000000") {
```

Comment:
I wonder if we could hide this logic and just import a constant that is named something like:
`isLocalStack` or something

Reply:
sounds good!